CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+# #develop#
```

Comment:
This should be replaced with the version

Reply:
Done.